engine/core/src/main/java/org/datacleaner/metamodel/datahub/DataHubDeleteBuilder.java

```diff
@@ -25,11 +26,22 @@
 import org.apache.metamodel.query.FilterItem;
 import org.apache.metamodel.schema.Table;
 
-public class DataHubDeleteBuilder extends AbstractRowDeletionBuilder{
+public class DataHubDeleteBuilder extends AbstractRowDeletionBuilder {
+    private static final String GR_ID_COLUMN_NAME = "gr_id";
+    private static final String SOURCE_ID_COLUMN_NAME = "source_id";
+    private static final String SOURCE_NAME_COLUMN_NAME = "source_name";
+
+    enum RecordType {
```

Comment:
I mean that the enum is a very big restriction around what DataHub can do (only work on person or organization). This restriction is unfortunately there in the backend but shouldn't slip out to the frontend IMO. We can just ask the backend to tell us what record types it has and keep that at the client. But not at compile time. It should be something we keep on request, making it possible for the server to improve and handle other record types.